subprojects/logging/src/test/groovy/org/gradle/internal/logging/console/ThrottlingOutputEventListenerTest.groovy

```diff
@@ -53,20 +64,28 @@ class ThrottlingOutputEventListenerTest extends OutputSpecification {
         renderer.onOutput(event3)
 
         then:
-        1 * listener.onOutput([event1] as ArrayList<OutputEvent>)
-        0 * _
+        interaction {
+            1 * listener.onOutput(event1)
+            expectRenderNowOnListener()
+            0 * _
+        }
 
         when:
         flush()
 
         then:
-        1 * listener.onOutput([event2, event3] as ArrayList<OutputEvent>)
-        0 * _
+        interaction {
+            1 * listener.onOutput(event2)
+            1 * listener.onOutput(event3)
+            expectRenderNowOnListener()
+            0 * _
+        }
 
         when:
         renderer.onOutput(event4)
 
         then:
+        _ * future._
         0 * _
```

Comment:
It sure seems like it as it explicitly prevents any change on the implementation (as we just did for the `RenderNowOutputEvent` as well as scheduling at fixed rate. I would be down to remove them and keep a more relaxed expectation. However, I may not see the whole picture here, and I was unsure about Gradle's mocking guidelines. Would you feel alright removing the `0 * _` expectation?